.github/workflows/tests-e2e.yml

```diff
@@ -135,10 +135,13 @@ jobs:
         run: npm run test:e2e:percy
         env:
           PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_E2E }}
+          WP_VERSION: ${{ matrix.wp }}
```

Comment:
This means we can't use 'latest' anymore..

Reply:
Can't the e2e test code just check for `WP_VERSION === 'latest'`?

Reply:
We will have to add a special case in `checkVersion` for latest.

Reply:
Not too big of a deal I think.